tvp_go19.go

```diff
@@ -20,12 +22,15 @@ const (
 )
 
 var (
-	ErrorEmptyTVPTypeName = errors.New("TypeName must not be empty")
-	ErrorTypeSlice        = errors.New("TVP must be slice type")
-	ErrorTypeSliceIsEmpty = errors.New("TVP mustn't be null value")
-	ErrorSkip             = errors.New("all fields mustn't skip")
-	ErrorObjectName       = errors.New("wrong tvp name")
-	ErrorWrongTyping      = errors.New("the number of elements in columnStr and tvpFieldIndexes do not align")
+	ErrorEmptyTVPTypeName     = errors.New("TypeName must not be empty")
```

Comment:
I'd like to unexport all of these errors. I can't see when I would want to switch on them.
```
var (
    errorEmptyTVPTypeName = ...
    ...
)
```

Before v1.0.0 is release, I really want to push the API to being as minimal as possible.

Reply:
I've done unexport the errors